### DISCUSSION TITLE
Clue Scrolls making clicks unresponsive

### DISCUSSION THREAD

**Discussion by jasonnrei** [runelite/runelite#16935] [Q&A]
Not sure if this is just a common issue having to wait for a clue scroll to go into your inventory, but it makes the next few clicks unresponsive after clicking a clue scroll while in the bank.

https://github.com/runelite/runelite/assets/142459637/93cc9d46-d7e1-4d58-8136-fd837b9bfde3

**Comment by geheur** [comment]
This is a jagex bug. https://github.com/geheur/bank-tag-custom-layouts/issues/71

**Reply by jasonnrei** [reply]
whats the infinity tab exactly?

**Reply by geheur** [reply]
The first one, with the "oo" aka infinity symbol.

**Reply by jasonnrei** [reply]
oh okay I see now, thank you :P
